.github/workflows/Nightly_Perf_Env_CI.yml

```diff
@@ -110,7 +110,8 @@ jobs:
               - 'vdbench_vm_scale'
               - 'clusterbuster'
               - 'bootstorm_vm_scale'
-              - 'windows_vm_scale'
+              - 'windows_vm_scale_windows_server_2019'
+              - 'windows_vm_scale_windows11'
```

Comment:
Ideally would be a separate option/environment variable to pick the desired OS, but I won't insist on that here.